2-mandatory/1-weather-report.js

```diff
@@ -13,13 +13,22 @@
 
 function getTemperatureReport(cities) {
     // TODO
+    let final = [];
+    for (let i = 0; i < cities.length; i++) {
```

Comment:
For used another method to loop. check it out, u may find it interesting:
    let tempDetailArr = []
    for (let cityName of cities) {
        const cityTemp = temperatureService(cityName)
        const tempDetail =  `The temperature in ${cityName} is ${cityTemp} degrees`
        tempDetailArr.push(tempDetail)
    }
    return tempDetailArr